### ISSUE TITLE
Easy way to make it work with the Interlink Elite?

### ISSUE THREAD

**Issue by NealisNeal** [taylorgoolsby/fpv-drone-issue-tracker#4]
Hello, I have a (really) old flight simulator transmitter that plugs into my pc via usb and I was wondering if there was an easy way to have the mod recognize it. Its called the Interlink Elite.

**Comment by taylorgoolsby** [comment]
Hi, can you verify that Windows detects your controller? Here is a screenshot walkthrough of how to find out: https://github.com/taylorgoolsby/fpv-drone-issue-tracker/issues/2#issuecomment-644421638

**Comment by NealisNeal** [comment]
Yes windows does recognize the controller, but when I click properties for the controller under "game controllers" the test screen shows no axis.
![Interlink Elite GC Test](https://user-images.githubusercontent.com/67359055/85618322-76af7480-b615-11ea-8b5e-e2166e75dcd7.PNG)

**Comment by taylorgoolsby** [comment]
If you go into the Settings tab, and then calibrate it, will that make the Test tab show the axis and buttons?

**Comment by NealisNeal** [comment]
It doesn't even give the option to calibrate it xD
![Interlink Elite GC Test2](https://user-images.githubusercontent.com/67359055/85619967-f1798f00-b617-11ea-9a49-a440ecf5f782.PNG)

This is a really old controller (like windows 7 stuff) and I don't think RealFlight has released updated drivers for it. I will look for more drivers for the thing in the meantime.

**Comment by taylorgoolsby** [comment]
I found a post of someone getting the Test tab to work with this controller: http://www.knifeedge.com/forums/index.php?threads/using-the-interlink-elite-with-other-games.31233/post-261857

I'm not sure why your's in particular isn't being detected in the same way. Could be a driver issue.

Another thing to try is to go into "Device Manager" in windows, and find "InterLink Elite" and uninstall the driver (with the controller unplugged). Then plug it back in, and let windows install default drivers.

**Comment by NealisNeal** [comment]
I just realized that on this computer I have not actually installed the RealFlight flight sim program, which will probably have something do with the controller calibration. Either way I will install it and report back if it changes anything.

**Comment by NealisNeal** [comment]
Alright, so i was able to install the program and doing that made the controller be recognized more, however minecraft does not seem to recognize it.

![Interlink Elite GC Test3](https://user-images.githubusercontent.com/67359055/85627086-d8c2a680-b622-11ea-8a0f-cde2398821f2.PNG)

This stupid controller has really weird axis and one of the nobs is the x rotation and the throttle is the z axis but that shouldn't matter if the in game axis are changeable. Im going to do a restart to see if thatll make minecraft recognize it.

**Comment by taylorgoolsby** [comment]
Nice! Almost there!

Try disconnecting your XBOX controller. The mod will detect only the first controller it sees, so when there are multiple controller's connected, you need to set the one you want to use as the default controller. I think in the Game Controllers window, you can right click on the controller you want, and set it as the default.

Or, to be extra sure, make your controller the only one plugged in.

**Comment by NealisNeal** [comment]
Awesome! After the restart I disconnected my XBOX Controller and it recognized it so I just had to change the axis and it works! I will just make sure my XBOX Controller is disconnected in the future.

To anyone who wants to use an Interlink Elite in the future the controls that work for me are: 
Throttle: CH3 Inverted
Roll: CH1
Pitch: CH2 Inverted
Yaw: CH 5
Custom Camera Angle: CH 4 (the "flaps" nob on the top right)
Arm: CH 7
Activate Custom Camera Angle: CH 9
